src/content/zero-knowledge-proofs.md

```diff
@@ -0,0 +1,178 @@
+
+Trusted setups require users to trust the integrity of everyone involved in the parameter-generation process, negating the ‘trustlessness’ principle underpinning blockchain designs. However, advanced research has led to the creation of ZK-STARKs, proving protocols that work with a non-trusted setup. 
+
+#### ZK-STARKs {#zk-starks}
+
+ZK-STARK is an acronym for **Zero-Knowledge Scalable Transparent Argument of Knowledge**. If you notice, a ZK-STARK is similar to a ZK-SNARK, except for two qualities:
+
+- **Scalable**: ZK-STARK is faster than ZK-SNARK at generating and verifying proofs when the size of the witness is larger. With STARK proofs, prover and verification times only slightly increase as the witness grows (SNARK prover and verifier times increase linearly with witness size). 
+
+- **Transparent**: ZK-STARK relies on publicly verifiable randomness to generate public parameters for proving and verification instead of a trusted setup. Thus, they are more transparent compared to ZK-SNARKs. 
+
+While ZK-STARK improves on ZK-SNARK, it has a significant drawback: it produces larger proofs. As a result, STARK proofs have higher verification overhead—although this arguably becomes less of an issue when generating proofs for large datasets. In this case, a ZK-STARK may be more cost-effective than a ZK-SNARK. 
+
+## Use-cases for zero-knowledge proofs {#use-cases-for-zero-knowledge-proofs}
+
+### Anonymous payments {#anonymous-payments} 
+
+Credit card payments are often visible to multiple parties, including the payments provider, your bank, and other interested parties (e.g., government authorities). While financial surveillance is good for curbing illegal financial activities, it poses risks for ordinary citizens. Such individuals may wish to keep details of their transactions away from prying eyes. 
+
+Cryptocurrencies were designed to provide a means for users to conduct private, peer-to-peer transactions. But cryptos only offer marginal improvements on financial privacy, as transaction details are publicly visible on the blockchain. Also, user identities are pseudonymous, not completely anonymous, allowing cutting-edge blockchain analysis tools to correlate wallet addresses with real-life identities. 
+
+However, continued research into financial privacy techniques led to the development of “privacy coins”, which are cryptocurrencies specifically designed for completely anonymous transactions. Privacy-focused blockchains, such as ZCash and Monero, shield transaction details, including sender/receiver addresses, asset type, quantity, and the transaction timeline. 
```

Comment:
Changing this because I don't think it is research chronology that explains the privacy/non-privacy coin dynamic. ZCash, Monero etc have been around for ages and it is not newness that holds them back. IMO, it is lack of demand combined with the illegality connotations of choosing privacy tools.

```suggestion
There are specific “privacy coins” designed for completely anonymous transactions. Privacy-focused blockchains, such as ZCash and Monero, shield transaction details, including sender/receiver addresses, asset type, quantity, and the transaction timeline.
```